ci/jobs/management.yml

```diff
@@ -5,6 +5,7 @@ jobs:
     passed:
     - management-dev
     trigger: true
+  - get: packer-egress-test-release
```

Comment:
this should have passed in mgmt-dev before being used in mgmt

Reply:
Ta, fixed.